src/planner/plan/append_simple.cpp

```diff
@@ -39,7 +42,37 @@ void Planner::appendCreateTable(const BoundStatement& statement, LogicalPlan& pl
     auto info = createTable.getInfo();
     auto op = make_shared<LogicalCreateTable>(info->copy(),
         statement.getStatementResult()->getSingleColumnExpr());
-    plan.setLastOperator(std::move(op));
+
+    // If it is CREATE NODE TABLE AS, then copy as well
+    if (createTable.hasCopyInfo()) {
+        auto dummySink = std::make_shared<LogicalDummySink>(std::move(op));
+        auto& querySource = createTable.getCopyInfo()->source->constCast<BoundQueryScanSource>();
+        auto queryPlan = getBestPlan(planQuery(*querySource.statement));
```

Comment:
Can you reuse `planCopyFrom`? I think that would make the code simpler.

Reply:
Done with https://github.com/kuzudb/kuzu/pull/5354/commits/89494098143e9a31c38af6915244e43d6231c1d5